constants/labels.js

```diff
@@ -0,0 +1,64 @@
+
+const prefixes = {
+    CUSTOMER: 'Customer',
+    BUG: 'Bug',
+    FEATURE: 'Feature'
+};
+
+const bugTypes = {
+    PRODUCTION: 'production',
+    DEVELOPMENT: 'development'
+}
+
+const featureTypes = {
+    NEW: 'new',
+    EXISTING: 'existing'
+}
+
+const getPrefix = (label) => (label || '').split(':')[0].trimRight();
+
+function getType (label) {
+    const labelType = (label || '').split(':');
+    return (labelType.length > 1) ? labelType[1].trimLeft() : '';
+}
+
+function getTitle (label) {
+    const prefix = getPrefix(label);
+    const type = getType(label);
+
+    switch(prefix) {
+        case prefixes.FEATURE:
+            switch (type) {
+                case featureTypes.EXISTING:
+                    return `\n\n## Improvements`;
+                case featureTypes.NEW:
+                    return `\n\n## New Features`;
+                default:
+                    return `\n\n## ${prefix}:${type} - No title preset`;
+            }
+        case prefixes.BUG:
+            switch (type) {
+                case bugTypes.PRODUCTION:
+                    return `\n\n## Bug fixes`;
+                default:
+                    return `\n\n## ${prefix}:${type} - No title preset`;
+            }
+        default:
+            return `\n\n## ${prefix} - No title preset`;
+    }
+}
+
+function checkSomeLabelEquals(labels, prefix, value) {
```

Comment:
I think this function is a bit confusing? AFAICS, this is checking if any label in `labels` matches a given prefix and value pair? I think would be clearer if use consistent naming, e.g. `labelGroup` and `labelType`?